src/ui/pages/work/WorkorderScheduling.js

```diff
@@ -41,6 +41,12 @@ const WorkorderScheduling = (props) => {
                 autocompleteHandler={WS.autocompleteEmployee}
             />
 
+            <EAMAutocomplete
+                {...register('schedgroup','assignedBy')}
+                barcodeScanner
+                autocompleteHandler={WS.autocompleteSupervisor}  //TODO: We require a new implementation of the autocomplete but for supervisors
```

Comment:
While there is no PR for the newly required ws for supervisors, this merge request should be marked as WIP. Once it is ready, it should explicitly reference the backend MR on which it relies.

Reply:
Here is the [MR](https://github.com/cern-eam/eam-light-backend/pull/107) for the revisors' autocomplete.